src/scrot_selection.c

```diff
@@ -331,9 +331,10 @@ bool scrotSelectionGetUserSel(struct SelectionRect *selectionRect)
                 scrotSelectionMotionDraw(rx, ry, ev.xbutton.x, ev.xbutton.y);
                 break;
             }
-            case KeyRelease:
-                /* ignore */
+            case DestroyNotify:
+                errx(EXIT_FAILURE, "received DestroyNotify event");
                 break;
+            case KeyRelease: /* ignore */
```

Comment:
Should just remove this case because it'll show up in the binary, put a comment mentioning ignoring KeyRelease is intentional if you want to.